spock/mcp/yggdrasil.py

```diff
@@ -7,21 +7,17 @@
     import urllib2 as request
     from urllib2 import HTTPError
 
+
 class YggAuth:
-    def __init__(self, 
-        client_token=None, 
-        access_token=None, 
-        username=None, 
-        password=None
-    ):
+    def __init__(self, client_token=None, access_token=None, username=None,
+                 password=None):
         self.username = username
         self.password = password
         self.client_token = client_token
 
-        self.access_token = None #validate needs self.access_token to exist
-        self.access_token = (
-            None if self.validate(access_token) else access_token
-        )
+        self.access_token = None  # validate needs self.access_token to exist
+        self.access_token = None if self.validate(access_token) else \
```

Comment:
I am afraid this doesn't worth a rewrite because I will be refactoring the file. (notice that it has a double assignment of self.access_token! WTF!)